quickstart/backend/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2025, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+SPDX-License-Identifier: 0BSD
+-->
+<configuration scan="false">
+
+    <!-- Include Spring Boot defaults (for profile-coloring, etc.) -->
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    <springProperty scope="context" name="LOG_FILE" source="logging.file.name"/>
+    <springProperty scope="context" name="LOG_PATH" source="logging.path"/>
+
+    <appender name="JSON_CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
+            <providers>
+                <timestamp/>
+                <logLevel/>
+                <threadName/>
+                <loggerName/>
+                <message/>
+                <keyValuePairs/>
+                <context/>
+                <version/>
+            </providers>
+        </encoder>
+    </appender>
+
+    <!-- Root logger set to OFF to prevent logging from any other package -->
+    <root level="OFF">
```

Comment:
This feels wrong to me. Why is this OFF and not WARN or at least ERROR? Are those levels really so noisy as to be a problem here?